src/SkyDrop.Droid/Helper/AndroidUtil.cs

```diff
@@ -111,9 +108,9 @@ public static string GetFileName(Context context, Android.Net.Uri uri)
 
         public static void OpenFileInBrowser(Activity context, SkyFile file)
         {
-            Toast.MakeText(context, $"Opening file {file.Filename}", ToastLength.Long).Show();
+            Toast.MakeText(context, $"Opening file {file.Filename}", ToastLength.Long)?.Show();
```

Comment:
Why would it be null?

Reply:
IDK but don't want to crash the app if it were